File: fibo.cpp
#include <iostream>
using namespace std;

void fibo(int lim, int t1 = 0, int t2 = 1)
{
  cout << "fibonaaci series: " << t1 << " " << t2;

  int nextTerm;

  for (int i = 3; i <= lim; i++)
  {
    nextTerm = t1 + t2;
    cout << " " << nextTerm;
    t1 = t2;
    t2 = nextTerm;
  }
}

int main()
{

  fibo(10);
  return 0;
}

File: Programs/Fibo.java
public class Fibo {
  public static void main(String[] args) {
    int t1 = 0, t2 = 1, t3, lim = 10;

    System.out.print("fibonaaci series: " + t1 + " " + t2);

    for (int i = 3; i <= lim; i++) {
      t3 = t1 + t2;
      System.out.print(" " + t3);
      t1 = t2;
      t2 = t3;
    }
  }
}
